packages/frontend-shared/cypress/support/component.ts

```diff
@@ -76,5 +76,13 @@ Cypress.on('uncaught:exception', (err) => !err.message.includes('ResizeObserver
 
 registerMountFn()
 addVueCommand()
-installCustomPercyCommand()
+installCustomPercyCommand({
+  isComponentTesting: true,
+  elementOverrides: {
+    'svg.animate-spin': ($el) => {
+      $el.attr('style', 'animation: none !important')
```

Comment:
Just wondering, do you have context on why we use `elementOverrides` as opposed to [percyCss](https://docs.percy.io/docs/percy-specific-css#snapshot-options--sdk-options)? percyCss seems a bit easier to work with than jquery, but they both get the job done!

Reply:
Omg forget about percyCSS, the [Percy media query option](https://docs.percy.io/docs/percy-specific-css#percy-css-media-query) is even slicker!!! Wonder why we're not using it?
<img width="836" alt="Screen Shot 2022-07-28 at 11 14 54 AM" src="https://user-images.githubusercontent.com/26726429/181608672-4ce2b594-b58b-4c48-b07c-9eca0409ad47.png">

Reply:
From what I understand, the reason for this is so that, in open mode, we get a Cypress snapshot of the exact DOM state that will be sent to Percy when you hover over the "skipping percy snapshot in interactive mode", which is a good way of previewing what Percy will render (though, that log doesn't seem to output in CT, which I have not investigated).